rust/connlib/clients/shared/src/messages.rs

```diff
@@ -297,6 +297,67 @@ mod test {
         assert_eq!(m, ingress_message);
     }
 
+    #[test]
```

Comment:
I would add more tests here if possible since they're cheap to add and give us some peace of mind.

I.e.

Test that new fields can contain `null`, positive/negative integers/floats, and other objects and arrays

Reply:
These tests are going to be very helpful when we're live and need to be confident in what's a breaking API change or not.